libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.spec.ts

```diff
@@ -0,0 +1,318 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { BrowserModule, By } from '@angular/platform-browser';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { expect, expectAsync } from '@skyux-sdk/testing';
+import { SkyIdService } from '@skyux/core';
+import { SkyPopoverHarness } from '@skyux/popovers/testing';
+import {
+  SkyTheme,
+  SkyThemeMode,
+  SkyThemeService,
+  SkyThemeSettings,
+  SkyThemeSettingsChange,
+} from '@skyux/theme';
+
+import { BehaviorSubject } from 'rxjs';
+
+import { SkyHelpInlineModule } from '../help-inline/help-inline.module';
+
+import { HelpInlineTestComponent } from './fixtures/help-inline.component.fixture';
+
+describe('Help inline component', () => {
+  async function checkAriaPropertiesAndAccessibility(
+    ariaLabel: string,
+    ariaControls: string | null,
+    ariaExpanded: string | null,
+  ): Promise<void> {
+    const helpInlineElement =
+      fixture.nativeElement.querySelector('.sky-help-inline');
+    expect(helpInlineElement?.getAttribute('aria-label')).toBe(ariaLabel);
+    expect(helpInlineElement?.getAttribute('aria-controls')).toBe(ariaControls);
+    expect(helpInlineElement?.getAttribute('aria-expanded')).toBe(ariaExpanded);
+    await expectAsync(fixture.nativeElement).toBeAccessible();
+  }
+
+  async function getPopoverTestHarness(): Promise<{
+    popoverHarness: SkyPopoverHarness;
+  }> {
+    const loader = TestbedHarnessEnvironment.documentRootLoader(fixture);
+    const popoverHarness = await loader.getHarness(SkyPopoverHarness);
+
+    return { popoverHarness };
+  }
+
+  let fixture: ComponentFixture<HelpInlineTestComponent>;
+  let component: HelpInlineTestComponent;
+  let debugElement: DebugElement;
+  let mockThemeSvc: { settingsChange: BehaviorSubject<SkyThemeSettingsChange> };
+
+  beforeEach(() => {
+    mockThemeSvc = {
+      settingsChange: new BehaviorSubject<SkyThemeSettingsChange>({
+        currentSettings: new SkyThemeSettings(
+          SkyTheme.presets.default,
+          SkyThemeMode.presets.light,
+        ),
+        previousSettings: undefined,
+      }),
+    };
+
+    TestBed.configureTestingModule({
+      imports: [
+        BrowserModule,
+        SkyHelpInlineModule,
+        NoopAnimationsModule,
+        HelpInlineTestComponent,
+      ],
+      providers: [{ provide: SkyThemeService, useValue: mockThemeSvc }],
+    });
+
+    fixture = TestBed.createComponent(HelpInlineTestComponent);
+    component = fixture.componentInstance as HelpInlineTestComponent;
+    debugElement = fixture.debugElement;
+
+    // Mock the ID service.
+    let uniqueId = 0;
```

Comment:
This should be pulled out of the `beforeEach` so that it doesn't get set to `0` again for each test.